hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -6446,4 +6446,11 @@
       frequently than this time, the client will give up waiting.
     </description>
   </property>
+  <property>
+    <name>dfs.observer.read.enable</name>
+    <value>true</value>
+    <description>
+      Enable observer read for client with router
```

Comment:
This is for all HDFS clients, right? Not just the routers.

I'd propose something like, "Enable the client to use observer NN for read operations. It RBF, it enables the routers to use the observer NN."

Reply:
I just realized setting this to false breaks direct reads to the observer namenodes.
I've add documentation to alert user of this. I'll investigate further.

Reply:
I no longer have this setting in the client. We may add it back in a future PR.